packages/odyssey-storybook/applitools.config.js

```diff
@@ -27,7 +27,10 @@ module.exports = {
     guidelinesVersion: "WCAG_2_1",
   },
   include({ name }) {
-    if (name === "Toast.Provider") return false;
+    if (["Toast.Provider"].indexOf(name) >= 0) {
```

Comment:
@jeffbelser-okta We didn't end up needing to hide any stories for `Banner`, but I thought it might be worth refactoring our setup to make VRT-hiding easier. Thoughts on this approach?

Reply:
yea lets move this list of excluded stories somewhere else - also since ES2015 you can use `Array.prototype.includes` rather than the explicit index comparison. So something like this:


```
const VRT_IGNORE = 'Toast.Provider Foo Bar Baz'.split('');
...

include ({ name }) {
  if (VRT_IGNORE.includes(name)) return false
  return true
}
```

Reply:
if we find that this list of excluded tests is relevant elsewhere we can further extract it into its own module.

We also don't need to maintain a list of strings either if we want to use more sophisticated scripting to match on other data or a convention similar to the `dont` one above